wait-for

```diff
@@ -141,6 +142,10 @@ while :; do
     TIMEOUT="${1#*=}"
     shift 1
     ;;
+    -v | --version)
+    echo 2.1.3
```

Comment:
Keeping this version in check should be automated, e.g. using https://github.com/google/semantic-release-replace-plugin.